aspnetcore/fundamentals/routing.md

```diff
@@ -30,7 +30,7 @@ Routing uses *routes* (implementations of `IRouter`) to:
 
 * generate URLs used in responses
 
-Generally an app has a single collection of routes. The route collection is processed in order. Requests look for a match in the route collection by . Responses use routing to generate URLs.
+Generally an app has a single collection of routes. The route collection is processed in order. Requests look for a match in the route collection by . Responses use routing to generate URLs (for example, for redirection).
```

Comment:
That's a pre-existing bug. I couldn't figure it out either, but I guess something along the lines of "by runing the XXXAsync method"? (I can't remember the method name right now.)